dGame/dInventory/Inventory.cpp

```diff
@@ -231,41 +231,42 @@ eInventoryType Inventory::FindInventoryTypeForLot(const LOT lot) {
 	const auto itemType = static_cast<eItemType>(itemComponent.itemType);
 
 	switch (itemType) {
-	case eItemType::ITEM_TYPE_BRICK:
+	case eItemType::BRICK:
 		return BRICKS;
 
-	case eItemType::ITEM_TYPE_BEHAVIOR:
+	case eItemType::BEHAVIOR:
 		return BEHAVIORS;
 
-	case eItemType::ITEM_TYPE_PROPERTY:
+	case eItemType::PROPERTY:
 		return PROPERTY_DEEDS;
 
-	case eItemType::ITEM_TYPE_MODEL:
-	case eItemType::ITEM_TYPE_VEHICLE:
-	case eItemType::ITEM_TYPE_LOOT_MODEL:
-	case eItemType::ITEM_TYPE_MOUNT:
+	case eItemType::MODEL:
+	case eItemType::VEHICLE:
+	case eItemType::LOOT_MODEL:
+	case eItemType::LUP_MODEL:
+	case eItemType::MOUNT:
 		return MODELS;
 
-	case eItemType::ITEM_TYPE_HAT:
-	case eItemType::ITEM_TYPE_HAIR:
-	case eItemType::ITEM_TYPE_NECK:
-	case eItemType::ITEM_TYPE_LEFT_HAND:
-	case eItemType::ITEM_TYPE_RIGHT_HAND:
-	case eItemType::ITEM_TYPE_LEGS:
-	case eItemType::ITEM_TYPE_LEFT_TRINKET:
-	case eItemType::ITEM_TYPE_RIGHT_TRINKET:
-	case eItemType::ITEM_TYPE_COLLECTIBLE:
-	case eItemType::ITEM_TYPE_CONSUMABLE:
-	case eItemType::ITEM_TYPE_CHEST:
-	case eItemType::ITEM_TYPE_EGG:
-	case eItemType::ITEM_TYPE_PET_FOOD:
-	case eItemType::ITEM_TYPE_PET_INVENTORY_ITEM:
-	case eItemType::ITEM_TYPE_PACKAGE:
-	case eItemType::ITEM_TYPE_CURRENCY:
```

Comment:
We should keep functional changes like this one out of this PR.

Reply:
I would agree, but:
1. There's no items with that type, so there is essentially no functional change
2. It's correcting the type to be named correctly, which it's a model, so goes in models

Reply:
will revert change if you really want me to though

Reply:
this change is fine since its a name correction and correcting the inventory an item of this type would go in